lib/ExtUtils/Liblist/Kid.pm

```diff
@@ -49,8 +49,10 @@ sub _unix_os2_ext {
     # $potential_libs
     # this is a rewrite of Andy Dougherty's extliblist in perl
 
+    require Text::ParseWords;
+
     my ( @searchpath );    # from "-L/path" entries in $potential_libs
-    my ( @libpath ) = split " ", $Config{'libpth'} || '';
```

Comment:
Are you sure this is correct? In my understanding, $Config{libpth} is (currently) never quoted, and treating it as if it is will not result in a correct end result.

Reply:
This was intended as protection for the future. The only way it can go wrong currently is if there are accidental quote-characters in there, which will cause huge problems already.

Reply:
If the encoding of libpth ever changes (to support spaces in paths), then we can easily (perl) version check to handle it.